packages/front-end/components/Features/ConditionInput.tsx

```diff
@@ -56,14 +57,11 @@ export default function ConditionInput(props: Props) {
   if (advanced || !attributes.size || !simpleAllowed) {
     return (
       <div className="mb-3">
-        <Field
+        <CodeTextArea
           label="Targeting Conditions"
-          containerClassName="mb-0"
-          textarea
-          minRows={1}
-          maxRows={12}
+          language="json"
           value={value}
-          onChange={(e) => setValue(e.target.value)}
+          setValue={setValue}
           helpText="JSON format using MongoDB query syntax"
```

Comment:
The "switch to simple mode" link looks out of place.  I think it might look better to add it to the `helpText` instead.

```tsx
helpText={(
  <div className="d-flex">
    <div>JSON format using MongoDB query syntax.</div>
    {simpleAllowed && attributes.size && (
      <div className="ml-auto">
        <a href="#" .../>
      </div>
    )}
  </div>
)}
```